### ISSUE TITLE
Allow overriding loader and parameters for efistub

### ISSUE THREAD

**Issue by m00nwtchr** [archlinux/archinstall#1105]
E.G. to be able to change the name of the loader to `<kernel-name>-signed.efi` for a Secure Boot Unified Kernel Image. (and empty kernel params, since they're embedded in the UKI)

**Comment by Torxed** [comment]
Just for my own understanding. calling it `-signed.efi` will not actually make it signed, you would have to sign it manually anyway. So naming it so might be causing confusion and this is best done outside of the installer. At least until we have created the secure-boot option in the installer.

**Comment by m00nwtchr** [comment]
Yes, i'm only asking for is to allow changing the name, the rest of the secure boot setup would be done outside of `archinstall` (since currently its not possible, unless you run `efibootmgr` yourself instead of using `archinstall`'s helper functions)
